install_config/configuring_sdn.adoc

```diff
@@ -211,6 +211,50 @@ Check VNIDs by running:
 $ oc get netnamespace
 ----
 
+[[migrating-between-sdn-plugins-networkpolicy]]
+=== Migrating from the ovs-multitenant Plugin to ovs-networkpolicy
+
+If you want to migrate from *ovs-multitenant* to *ovs-networkpolicy*, you will first need
+to make sure that every namespace has a unique `NetID`; if you have previously
+xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined projects
+together] or xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made
+projects global], you will need to undo that before switching to the *ovs-networkpolicy*
+plugin, or NetworkPolicy objects may not function correctly.
+
+A helper script is available that will take care of this, and also create NetworkPolicy
+objects to isolate previously-isolated namespaces, and allow connections between
+previously-joined namespaces.
+
+. Download https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh
++
+----
+$ curl https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh
+$ chmod a+x migrate-network-policy.sh
+----
+. While still running the *ovs-multitenant* plugin (that is, before running any of the
+migration steps above), run that script as a user with cluster administrator rights on
+the OpenShift cluster
++
+----
+$ ./migrate-network-policy.sh
+----
+
+After running the script, every project will now be fully isolated from every other
+project, so connection attempts between pods in different projects will fail until you
+complete the migration to the *ovs-networkpolicy* plugin.
+
+The script will also set up
+xref:../admin_guide/managing_networking.adoc#admin-guide-networking-networkpolicy-setting-default[default
+NetworkPolicy objects] to be created in any newly-created projects so that they will have
+the same isolation behavior as existing projects (pods will allow connections only from
+other pods in the same project, or from pods in the "default" namespace).
+
+(If something goes wrong and you need to revert back to the *ovs-multitenant* plugin, there
```

Comment:
```
[NOTE]
====
In case of script failures or other errors, switch to the *ovs-multitenant*
plugin and use the link:https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/unmigrate-network-policy.sh[un-migration script] to revert back. This script undo the changes made by the migration plugin, and
re-join previously-joined projects.
====
```

Are there any instructions available on switching back to the  *ovs-multitenant* plugin?

Reply:
> Are there any instructions available on switching back to the ovs-multitenant plugin?

Actually, that section was supposed to be about exactly that; how to switch back to ovs-multitenant if you wanted to do that later. But your comment about "script failures or other errors" is also something we should mention, so I updated the text to cover both cases.